### DISCUSSION TITLE
theme.css created with visual theme app

### DISCUSSION THREAD

**Discussion by robertdmartin** [primefaces/community#698] [PrimeNG]
The theme.css file produced with the legacy theme builder references font files that are local to where the file is in the project. I put it @ /src/theme/theme.css ... naturally the build fails b/c it cannot fint the fonts. Where can I get these fonts? I do not see them anywhere in the distribution ...

` /airsyn-ng/src/theme/theme.css:1:1202: Can't resolve './fonts/poppins-v15-latin-ext_latin-300.woff' in '/airsyn-ng/src/theme'`

?

**Comment by cetincakiroglu** [comment]
Hi,

Thanks for the feedback, we'll check and try to find root cause of the issue.
